doc/Language/control.pod6

```diff
@@ -535,6 +535,27 @@ Assigning C<Empty> will effectively undefine an Array, using C<for> over an
 undefined array will not even enter the loop, as shown, effectively behaving in
 the same way as above when C<Empty> was used directly.
 
+
+With C<hyper> and C<race> the for loop is potentially iterated in parallel.
+See also the documentaion for C<hyper> and C<race> in class L<Map|/type/Map>.
+
+
+=for code
+my $primes_h = hyper for ^10_000 -> $number { $number if $number.is-prime };
+say $primes_h.elems;
+say $primes_h.tail: 5; # OUTPUT: (9931 9941 9949 9967 9973)
+
+with C<hyper> the order of elements is preserved.
+
+=for code
+my $primes_r = race  for ^10_000 -> $number { $number if $number.is-prime };
+say $primes_r.elems;
+say $primes_r.tail: 5; # OUTPUT: (9871 9883 9887 9901 9907)
```

Comment:
Probably better to replace the sample output with something like "Will print last 5 values of stream without particular order". On the other hand, it will duplicate the line below... Maybe it's better to just remove this line. Or... "Output can be: (9871 9883 9887 9901 9907)".